### ISSUE TITLE
lockFile is global for all users

### ISSUE THREAD

**Issue by izahn** [kapitainsky/RcloneBrowser#162]
I found that if more than one user in a multi-user linux environment tries to use rclone-browser at the same time we get  message saying that "Rclone Browser is already running. Only one instance is allowed.".

It looks like https://github.com/kapitainsky/RcloneBrowser/blob/45bf7411839b0919c488a3e3a0a5272b50c9fc7b/src/main.cpp#L201 implements a global lock instead of a per-user lock. Is that intended? I've patched this up locally with 

```
  char* lockLocalUserName = std::getenv("USER");
  QLockFile lockFile(tmpDir + "/." + lockLocalUserName + "RcloneBrowser_4q6RgLs2RpbJA.lock");
```
but I don't know if this is the right approach, or if the global lock was intentional.

**Comment by kapitainsky** [comment]
Yes this was my intention but indeed per user would be maybe be better - I will merge it

Thx!
